.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "benches/quilc"]
+	path = benches/quilc
+	url = git@github.com:quil-lang/quilc.git
```

Comment:
Not a fan of submodules personally, I find they tend to be hard to maintain. That said—I don't think there's an easier way to get `quilc` other than Docker and Docker would make benchmarking harder 🤷.

Reply:
Yea, generally I agree.. avoid if possible. We can come back to this later if it starts to make things gross!